sources/osg/Matrix.js

```diff
@@ -1277,5 +1557,7 @@ define( [
         }
     };
 
+    Matrix.IDENTITY = [ 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 0.0, 1.0 ];
```

Comment:
I tought that was the norm ?
(from other class static const in other obj like Texture.FILTER, etc.)